teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission: parent_permission, name: name)
```

Comment:
- [ ] Kindly remove the `id` parameter in the `initialize` method of the `Teacher` class and from `super` as well as explained earlier in this comment.